clients/docs/c.rst

```diff
@@ -0,0 +1,156 @@
+.. _client-examples-c:
+
+C
+===
+
+
+In this tutorial, you will run a C client application that produces messages to
+and consumes messages from an |ak-tm| cluster.
+
+.. include:: includes/client-example-overview.rst
+
+
+Prerequisites
+-------------
+
+Client
+~~~~~~
+
+-  `librdkafka <https://github.com/edenhill/librdkafka>`__ installed on
+   your machine. See the `librdkafka installation
+   instructions <https://github.com/edenhill/librdkafka/blob/master/README.md#instructions>`__.
+
+-  Create a local file (for example, at ``$HOME/.confluent/librdkafka.config``)
+   with configuration parameters to connect to your Kafka cluster, which can be
+   on your local host, `Confluent Cloud
+   <https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud>`__,
+   or any other cluster. Follow `these detailed instructions
+   <https://github.com/confluentinc/configuration-templates/tree/master/README.md>`__
+   to properly create this file.
+
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. Clone the `confluentinc/examples GitHub repository
+   <https://github.com/confluentinc/examples>`__ and check out the
+   :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+      git clone https://github.com/confluentinc/examples
+      cd examples
+      git checkout |release|-post
+
+#. Change directory to the example for C.
+
+   .. code-block:: bash
+
+      cd clients/cloud/c/
+
+#. .. include:: includes/client-example-create-file.rst
+
+
+Basic Producer and Consumer
+----------------------------
+
+In this example, the producer writes JSON data to a topic in your Kafka cluster.
+Each record has a key representing a username (for example, ``alice``) and a
+value of a count, in JSON format (for example, ``{"count": 0}``). The consumer
+reads the same topic and keeps a rolling sum of the counts as it processes each
+record.
+
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+#. Run the producer, passing in arguments for:
+
+   - the local file with configuration parameters to connect to your Kafka
```

Comment:
For this example, we need to switch the order of these two bullet points (first `the topic name`, second `the local file...`) to match the code

Reply:
updated